### PR TITLE
Fix UG issues

### LINKED ISSUE #268: Fix UG issues
* Fix grammar
* Fix command table summary links not working

### PR DESCRIPTION
**Pull request by KrashKart**
* Fixes #268

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import seedu.address.model.person.Person;
 
 /**
- * Deletes a person identified using it's displayed index from the campus connect.
+ * Deletes a person identified using it's displayed index from CampusConnect.S
```
Comment (CYX22222003) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java:15]: I believe the S at the end is a typo.

**Review by CYX22222003 — COMMENTED** [review]
LGTM. Minor modifications needed

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (KrashKart) [reply src/main/java/seedu/address/logic/commands/DeleteCommand.java:15]: Good catch!

**Review by CYX22222003 — APPROVED** [review]
LGTM. This should be out last update on features

**Pull request merged**